packages/grid/src/vaadin-grid-keyboard-navigation-mixin.js

```diff
@@ -873,8 +865,6 @@ export const KeyboardNavigationMixin = (superClass) =>
           this._focusedCell = null;
         }
       }
-
-      this._detectFocusedItemIndex(e);
```

Comment:
Is this safe tor remove?

Reply:
I mean that `_detectFocusedItemIndex` was previously called here even if the logic didn't enter the if block. Maybe that was intentional for some reason.

Reply:
But there was pretty much the same condition inside:

```js
const { section, row } = this._getGridEventLocation(e);
if (section === this.$.items) {
  this._focusedItemIndex = row.index;
}
```

Reply:
I can extract this change in a separate PR by the way.